portfolio/src/main/webapp/index.html

```diff
@@ -42,37 +39,35 @@
         </button>
         <div class="collapse navbar-collapse" id="navbarResponsive">
           <ul class="navbar-nav text-uppercase ml-auto">
-            <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#services">Services</a></li>
-            <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#portfolio">Portfolio</a></li>
-            <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#about">About</a></li>
-            <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#team">Team</a></li>
-            <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#contact">Contact</a></li>
+            <li class="nav-item"><a href="index.html">Home</a></li>
+            <li class="nav-item"><a href="statistics.html">Statistics</a></li>
+            <li class="nav-item"><a href="about.html">About Us</a></li>
```

Comment:
Maybe the reason it isn't behaving like we wanted to earlier is because it doesn't include the 'js-scroll-trigger' class?

Reply:
I tried adding the nav-link and js-scroll-trigger, but the formatting was still not right. Not sure why though.